src/Certify.Providers/DNS/AcmeDns/AcmeDns/AcmeDns.csproj

```diff
@@ -8,6 +8,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\..\..\..\Certify.Models\Certify.Models.csproj" />
+    <ProjectReference Include="..\..\..\..\Certify.Shared\Certify.Shared.Core.csproj" />
```

Comment:
I moved the call to `Util.GetAppDataFolder()` from the class-specific initialization code that used to be in `ChallengeProvider` to the constructor. We could duplicate that function in the in this project instead of referencing that one...